pyrokinetics/dataset_wrapper.py

```diff
@@ -155,12 +155,18 @@ def __repr__(self) -> str:
         )
         return my_repr
 
-    # class: dataset_wrapper
     def __contains__(self, name: str) -> bool:
+        """Redirect ``x in y`` calls to the inner dataset"""
         return name in self.data
 
     def to_netcdf(self, *args, **kwargs) -> None:
-        """Writes self.data to disk. Forwards all args to xarray.Dataset.to_netcdf."""
+        """
+        Writes self.data to disk. Forwards all args to xarray.Dataset.to_netcdf.
+        Complex data is expanded out into float arrays of shape ``[dims..., 2]``.
+        """
+        # TODO Expand complex numbers selectively
```

Comment:
We could probably check if any of the data is complex and only convert to `ReIm` if its there. Would stop things like `Equilibrium` and `Kinetics` having this issue

Reply:
Sorted. I thought this would be a really annoying problem to solve, so I thought it best to put it off until a future PR, but it turned out to be much easier than expected.